k8s/k8s_descriptor-poz-dev.yaml

```diff
@@ -0,0 +1,207 @@
+      - name: tmp-dir
+        emptyDir: {}
+      initContainers:
+      - name: init-vault
+        image: artifactory.wikia-inc.com/ops/init-vault:0.0.36
+        args:
+        - "STORAGE_ACCESS_KEY=secret/app/dev/vignette/ceph_tokens_poz.access_key"
+        - "STORAGE_SECRET_KEY=secret/app/dev/vignette/ceph_tokens_poz.secret_key"
+        env:
+        - name: SERVICE_ACCOUNT
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.serviceAccountName
+        - name: K8S_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: VAULT_ADDR
+          value: active.vault.service.poz.consul:8200
+        - name: ENV
+          value: dev
+        volumeMounts:
+        - name: secrets-dir
+          mountPath: /var/lib/secrets
+      containers:
+      - name: vignette-tmp-cleaner
+        image: alpine:3.8
+        volumeMounts:
+        - name: tmp-dir
+          mountPath: /tmp-vignette
+        command:  # remove files older than 10 minutes
+         - /bin/sh
+         - -c
+         - "while true; do find '/tmp-vignette' -type f -mmin +10 -exec /bin/rm {} \\; 2> /dev/null; sleep 300; done"
+      - name: vignette
+        image: ${IMAGE_PATH}
+        # required for vault secrets to load
+        lifecycle:
+          preStop:
+            exec:
+              command: ["sleep", "3"]
+        env:
+        - name: HOST
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: KUBERNETES_POD
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: INSTANCE_ID
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: STORAGE_ENDPOINT
+          value: http://dfs
+        - name: STORAGE_PROXY
+          value: dev.dfs.service.poz-dev.consul
+        - name: STORAGE_PROXY_PORT
+          value: "80"
+        - name: VIGNETTE_TEMP_FILE_LOCATION   # temporary vignette files
+          value: /tmp-vignette
+        - name: MAGICK_TEMPORARY_PATH   # temporary ImageMagick files
+          value: /tmp-vignette
+        - name: LOGGER_TYPE
+          value: syslog
+        - name: LOGGER_FILE_OUTPUT
+          value: /tmp/vignette.log
+        - name: LOGGER_APPLICATION
+          value: vignette
+        - name: IMAGEMAGICK_BASE
+          value: /usr
+        - name: VIGNETTE_THUMBNAIL_BIN
+          value: /thumbnail
+        - name: VIGNETTE_BROKEN_IMAGE
+          value: /public/brokenImage.jpg
+        - name: GETOPT
+          value: /bin/getopt
+        - name: TIMEOUT
+          value: timeout
+        - name: TIMEOUT_TIME
+          value: "-t 30"
+        - name: GIF2WEBP
+          value: /usr/bin/gif2webp
+        - name: WIKIA_ENVIRONMENT
+          value: dev
+        - name: WIKIA_DATACENTER
+          value: poz
+        - name: PERFMONITORING_HOST
+          value: prod.app-metrics-etl.service.sjc.consul
```

Comment:
Are you sure this should be `prod`?

Reply:
yes, there is a single host for cheetah, it just listens on two ports: dev and prod

Reply:
ok 👍